composer.json

```
@@ -1,5 +1,5 @@
 {
-    "name": "lexpress/doctrine1",
+    "name": "fos1/doctrine1",
```

I recommend to avoid abbreviation here even if legacy packages use this short vendor name.

```suggestion
    "name": "friendsofsymfony1/doctrine1",
```

---

I like the idea, should we propose to use this "namespace" for all projects related to SF1? wdyt?

---

Only for them managed by friends of symfony1.

Doing this will maybe help to remove a confusion.